spec/queries/events_query_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EventsQuery, type: :query do
+  subject(:events_query) { described_class.new(organization:, pagination:, filters:) }
+
+  let(:organization) { create(:organization) }
+  let(:pagination) { nil }
+  let(:filters) { {} }
+
+  let(:event) { create(:event, timestamp: 1.days.ago.to_date, organization:) }
+
+  before { event }
```

Comment:
It would be great if you could create a few records and use them to check filters behave as expected with some events included in the result but not all of them.